pkg/controller/registry/resolver/resolver_test.go

```diff
@@ -144,7 +145,7 @@ func TestSolveOperators_WithSystemConstraints(t *testing.T) {
 			expectedOperators:         nil,
 			csvs:                      nil,
 			subs:                      []*v1alpha1.Subscription{packageASub},
-			err:                       "packageA is not white listed",
+			err:                       []string{"packageA is not white listed", "packageB is not white listed"},
```

Comment:
I believe that it makes more sense to improve this test by whitelisting the other packages:

```go
systemConstraintsProvider: whiteListConstraintProvider(packageB, packageC, packageD),
```
If you disagree, at least update the title of the test to more accurately reflect the next test.

Reply:
I agree.  It makes sense.  I'll put the suggested changes.